src/jit/compiler.h

```diff
@@ -1164,7 +1164,13 @@ struct fgArgTabEntry
     regNumber otherRegNum; // The (second) register to use when passing this argument.
 
     SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
-#endif // defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
+#elif defined(_TARGET_X86_)
+    __declspec(property(get = getIsStruct)) bool isStruct;
```

Comment:
If this is a brand new member, can we please not use a `__declspec(property)`? I'd prefer to be able to tell when a member access is a method call (and not a simple field load).